### ISSUE TITLE
Fix config builder

### ISSUE THREAD

**Issue by vouillon** [Thibaut-Gudin/ocaml-mixpanel#2]
I think the type of `config` should be:
```ocaml
val config :
  ?api_host:string ->
  ?app_host:string ->
  ?autotrack:bool ->
  ?cdn:string ->
  ?cross_subdomain_cookie:bool ->
  ?persistence:string ->
  ?persistence_name:string ->
  ?cookie_name:string ->
  ?loaded:(Ojs.t -> unit) ->
  ?store_google:bool ->
  ?save_referrer:bool ->
  ?test:bool ->
  ?verbose:bool ->
  ?img:bool ->
  ?track_links_timeout:int ->
  ?cookie_expiration:int ->
  ?upgrade:bool ->
  ?disable_persistence:bool ->
  ?disable_cookie:bool ->
  ?secure_cookie:bool ->
  ?ip:bool ->
  (*Check if PropertyBlacklist realy is a string list...*)
  ?property_blacklist:string list ->
  unit ->
  config
  [@@js.builder][@@js.verbatim_names]
```
(See https://github.com/LexiFi/gen_js_api/blob/master/NAMING.md)

**Comment by Thibaut-Gudin** [comment]
I applied this correction to this commit (I haven't tried if it works yet):
https://github.com/besport/ocaml-mixpanel/pull/1/commits/76ea0eed44ab0e76ac792645e6e12d1d014fc021

**Comment by ilankri** [comment]
Fixed by #1. Thanks!
